pages/docs/tutorials/kafka/configure-kafka-avro.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Describe Kafka message payload using Avro Schema
+description: Explore configuring AsyncAPI document for Kafka messages with Avro Schema.
+weight: 30
+---
+
+## Introduction
+
+The [previous tutorial](/docs/tutorials/kafka) taught you about writing an AsyncAPI document for Kafka messages using the default schema. This tutorial will teach you to write the same document using Avro Schema.
```

Comment:
Please improve the anchor text by actually writing the **title of that previous tutorial** 😸

Reply:
Done! thanks for pointing it out